addons/account_peppol/models/account_edi_proxy_user.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo import fields, models, _
+from odoo.addons.account_edi_proxy_client.models.account_edi_proxy_user import AccountEdiProxyError
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountEdiProxyClientUser(models.Model):
+    _inherit = 'account_edi_proxy_client.user'
+
+    proxy_type = fields.Selection(
+        selection_add=[('peppol', 'PEPPOL')],
+    )
+
+    def _get_proxy_urls(self):
+        urls = super()._get_proxy_urls()
+        urls['peppol'] = {
+            'prod': '',
+            'test': 'https://l10n-pe-edi.test.odoo.com',
+        }
+        return urls
+
+    def _get_server_url(self, proxy_type=None, edi_mode=None):
+        proxy_type = proxy_type or self.proxy_type
+        if not proxy_type == 'peppol':
+            return super()._get_server_url(proxy_type, edi_mode)
+
+        peppol_param = self.env['ir.config_parameter'].sudo().get_param(
+            'account_peppol.settings.view', False
```

Comment:
why `settings.view`?

Reply:
because it was originally created to show/hide things in the settings view based on this param and then reused here. Could be `account_peppol.test.mode` or something like that?

Reply:
yes, sounds better

Reply:
changed to `account_peppol.edi.mode`